client/src/app/auth/services/auth.service.ts

```diff
@@ -16,21 +17,21 @@ export class AuthService {
       })
       .pipe(
         tap((res: any) => {
-          this.setToken(res);
+          this.setUserData(res);
         })
       );
   }
 
-  private setToken(data: any): void {
-    localStorage.setItem('token', data.token);
+  private setUserData(userData: any): void {
```

Comment:
avoid using any, use interface

Reply:
I see any

Reply:
it's old, now it is changed in new commit